### DISCUSSION TITLE
What is this project about?

### DISCUSSION THREAD

**Discussion by yadyeh** [Investiway/investiway-front#14] [Q&A] [answered]
Can you talk about this project in detail? What can it do for me?

**Comment by heydayle** [comment, marked as answer]
> Can you talk about this project in detail? What can it do for me?

Investiway is a personal finance management.
It's a web application that allows users to keep track of their monthly and weekly expenses, plan their spending for the next month, add new spending categories, and set monthly recurring expenses. The application also provides advice for spending based on user history and allows users to create notes with priorities.

read more [here](https://github.com/Investiway/investiway-front/blob/main/README.md)
